requirements-dev.txt

```diff
@@ -3,11 +3,13 @@ jinja2
 numpy
 onnx
 onnxmltools
+packaging
 pandas
 protobuf
 pytest
 pytest-cov
 scikit-learn
 scipy
 six
+sympy
 wheel
```

Comment:
If we're adding sympy for the symbolic shape inferencing, should we add cntk so the gen.py dependencies are satisfied?

Reply:
I'd prefer to just have onnx related components. CNTK/TensorFlow/Keras should be added only at the place they are needed. Is that reasonable?

---
In reply to: [312431236](https://github.com/microsoft/onnxruntime/pull/881#discussion_r312431236) [](ancestors = 312431236)

Reply:
Seems reasonable. So scikit-learn shouldn't be in this list based on that right?  

---
In reply to: [312634525](https://github.com/microsoft/onnxruntime/pull/881#discussion_r312634525) [](ancestors = 312634525,312431236)

Reply:
Not sure about scikit-learn, I hope we could address that in a separate PR.

---
In reply to: [312762594](https://github.com/microsoft/onnxruntime/pull/881#discussion_r312762594) [](ancestors = 312762594,312634525,312431236)